rest/rest-client/src/test/java/functionaltests/SchedulerClientTest.java

```diff
@@ -473,6 +477,21 @@ public JobState getSubmittedJob() {
             }
         }
 
+        public UserIdentification getLastUserUpdate() {
+            System.out.println("getLastUserUpdate");
+            synchronized (this) {
+                if (userIdentificationStack.isEmpty()) {
+                    System.out.println("Stack is empty");
+                    try {
+                        System.out.println("wait");
```

Comment:
Why not use the logging system instead of sysout ?

Reply:
Fair point! tbh this is just a copy-paste from the above method (not a valid justification, I know ;) )

Reply:
I think it will make this message be displayed at everybuild, using a logger will only make it visible when you use --info or --debug flag... again, your call ;-)